book/src/faq.md

```diff
@@ -96,20 +96,22 @@ when a change is signaled, primarily to minimize latency and optimize client req
 By default, if your service does not emit change notifications, the property values will not
 update accordingly. 
 
+
```

Comment:
redundant change. Please go through every line of your commits before pushing (or after, if you forget). Otherwise, it waste reviewers time.

Reply:
I don't understand what's being discussed here I'll review the commit again when I'm on my machine to understand what's the redundancy.